bridge_adaptivity/module/static/module/js/module.js

```diff
@@ -360,9 +360,12 @@
         const modalSuccessListener = (url) => {
             return result => {
                 let modal = $('#modal-wrapper');
-                if (result.status !== undefined && result.status === 'ok' && result.collection_slug !== undefined) {
-                    var new_url = new URL(window.location.href);
+                var new_url = new URL(window.location.href);
```

Comment:
`let` definition for the `new_url` variable will be faire enough.

Reply:
Done